.github/workflows/msrv.yml

```diff
@@ -26,4 +26,4 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: check
-        args: --features std,serde,lax_deserialize
+        args: --all-features
```

Comment:
Not sure if you want to run the same suite of tests on the MSRV or not.

Reply:
Yeah. There used to be a feature that only worked on Nightly, but that has been removed.